src/ui/SearchInterface/SearchInterface.ts

```diff
@@ -437,6 +438,12 @@ export class SearchInterface extends RootComponent implements IComponentBindings
     Assert.exists(element);
     Assert.exists(this.options);
 
+    new InitializationPlaceholder(element, {
```

Comment:
So the InitializationPlaceholder gets created/executed 2 times now ?

Can we do it only once here ?

Reply:
Since I had to move part of the Placeholder logic inside the SearchInterface class, I was afraid that the "search interface" placeholder wouldn't do much anymore. With the options I passed to the constructor, the 2 calls shouldn't step on each others.

I'll try to move everything to the SearchInterface to see if it makes any difference.